### PR TITLE
fix: adjust header on mobile

### PR DESCRIPTION
**Pull request by Raduc4**

### CODE REVIEW

File: sections/header/header.section.tsx [hunk 1]
```diff
@@ -5,25 +5,15 @@ import { ReactComponent as MenuIcon } from './assets/menu';
 import { ReactComponent as LogoIcon } from './assets/Logo';
 import { Container } from '../../components/grid';
 
-const MobileLinks: React.FC = () => {
-  const [open, setOpen] = useState(false);
+interface props {
```
Comment (iamandrewluca) [inline sections/header/header.section.tsx:8]:
```suggestion
interface Props {
```

File: sections/header/header.section.tsx [hunk 2]
```diff
@@ -5,25 +5,15 @@ import { ReactComponent as MenuIcon } from './assets/menu';
 import { ReactComponent as LogoIcon } from './assets/Logo';
 import { Container } from '../../components/grid';
 
-const MobileLinks: React.FC = () => {
-  const [open, setOpen] = useState(false);
+interface Props {
+  open: boolean;
+}
 
+const MobileLinks: React.FC<props> = ({ open }) => {
```
Comment (iamandrewluca) [inline sections/header/header.section.tsx:12]: wai ce faci ))

```suggestion
const MobileLinks: React.FC<Props> = ({ open }) => {
```

Comment (Raduc4) [reply sections/header/header.section.tsx:12]: )) magie neagra

**Pull request merged**